### DISCUSSION TITLE
能否查看 envoy 配置内容

### DISCUSSION THREAD

**Discussion by nlg521** [higress-group/higress#744] [Q&A] [answered]
有没有一种方式查看 envoy 内的路由规则设置，主要用于排查问题时，查看路由规则设置的是否正确。

**Comment by CH3CHO** [comment, marked as answer]
可以使用 `hgctl gateway-config` 命令。

hgctl 工具的安装方法可以查看文档：https://higress.io/zh-cn/docs/ops/hgctl

**Reply by nlg521** [reply]
谢谢
